region/lowernorfair/west.json

```diff
@@ -1628,6 +1628,18 @@
                     {"heatFrames": 400}
                   ],
                   "note": "Position yourself in the door way, then run and jump. Makes it possible to walljump up."
+                },
+                {
+                  "name": "Screw Attack Room Doorway CWJ",
+                  "notable": true,
+                  "requires": [
+                    "h_heatProof",
+                    "HiJump",
+                    "canCWJ",
+                    "canStationarySpinJump",
+                    "canInsaneWallJump"
```

Comment:
I think this needs h_canDestroyBombWalls (Screw/PB/Morph bombs - tho morph bombs opens IBJ)

Reply:
This node requires that the above obstacle already has been cleared